release-notes/topics/operator-220.adoc

```diff
@@ -0,0 +1,14 @@
+[[operator-220-intro]]
+= Automation Platform Operator 2.2
+
+The Ansible Automation Platform Operator provides cloud-native, push-button deployment of new Ansible Automation Platform instances in your OpenShift environment.
+
+.Deprecated functionality
```

Comment:
I wonder if we might want to call this "Deprecated Parameters".  Not sure the best name, but no functionality is lost here, it's actually expanded, just under a different parameter name.  

Not a big deal either way, just a nitpick.

Reply:
Sounds like this is the generic title that will be included for future release notes too, so I can definitely see the use in keeping it general to fit other types of changes/deprecations.

Reply:
Yes, this is standard language across Red Hat and so I will leave it as is for now. Thanks!